nifty-client/src/main/java/com/facebook/nifty/client/FramedClientConnector.java

```diff
@@ -93,6 +93,9 @@ public ChannelPipeline getPipeline()
                                 LENGTH_FIELD_LENGTH,
                                 LENGTH_ADJUSTMENT,
                                 INITIAL_BYTES_TO_STRIP));
+                if (clientConfig.sslClientConfiguration() != null) {
+                    cp.addFirst("ssl", clientConfig.sslClientConfiguration().initializeHandler());
```

Comment:
the timeout handler code unfortunately adds stuff as first, so that ends up not working with ssl